src/views/SearchPage.vue

```diff
@@ -109,6 +109,16 @@ export default {
     }
   },
   methods: {
+    showMore(facet, facetName) {
+      if (facet.displayingAll) {
+        return;
+      }
```

Comment:
this should be unnecessary I think. the button should simply not show if displayingAll is true..